Net.DDP.Client/JsonDeserializeHelper.cs

```diff
@@ -3,141 +3,147 @@
 using System.Dynamic;
 using System.Linq;
 using Newtonsoft.Json.Linq;
+using System.Diagnostics;
+using System;
 
 namespace Net.DDP.Client
 {
-    internal class JsonDeserializeHelper
-    {
-        private readonly IDataSubscriber _subscriber;
-
-        public JsonDeserializeHelper(IDataSubscriber subscriber)
-        {
-            _subscriber = subscriber;
-        }
-
-        internal void Deserialize(string jsonItem)
-        {
-            JObject jObj = JObject.Parse(jsonItem);
-
-            if (jObj[DDPClient.DDP_PROPS_ERROR] != null || jObj[DDPClient.DDP_PROPS_MESSAGE] != null && jObj[DDPClient.DDP_PROPS_MESSAGE].ToString() == "error")
-                HandleError(jObj[DDPClient.DDP_PROPS_ERROR] ?? jObj);
-            else if (jObj[DDPClient.DDP_PROPS_SESSION] != null)
-                HandleConnected(jObj);
-            else if (jObj[DDPClient.DDP_PROPS_MESSAGE] != null)
```

Comment:
This was the main offending line: a result message *does* contain a message field, so controls incorrectly falls into this branch of the conditional, and therefore the parsing of the result message never happens.